src/client/components/EntryContent.jsx

```diff
@@ -3,8 +3,56 @@ import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import moment from 'moment';
 
-const Div = styled.div`
+// TODO: Add report and helpful button div
+const EntryContent = ({ review }) => {
+  const { date, foodScore, serviceScore, ambienceScore, overallScore } = review;
+  const text = review.review;
```

Comment:
Have you considered updating your DB schema so that you can just destructure this directly?